wrench-app/src/main/java/com/izettle/wrench/dialogs/scope/ScopeFragmentViewModel.kt

```diff
@@ -27,23 +28,27 @@ constructor(private val scopeDao: WrenchScopeDao) : ViewModel() {
     }
 
     internal fun selectScope(wrenchScope: WrenchScope) {
-        wrenchScope.timeStamp = Date()
-        scopeDao.update(wrenchScope)
+        GlobalScope.launch {
+            wrenchScope.timeStamp = Date()
+            scopeDao.update(wrenchScope)
+        }
     }
 
     @WorkerThread
```

Comment:
These annotations are not accurate any more

Reply:
👍 removed! good catch!